### PR TITLE
switch links from druid.io to druid.apache.org

### PR DESCRIPTION
**Pull request by clintropolis**
Does not include links to `static.druid.io`

### CODE REVIEW

File: docs/content/ingestion/hadoop-vs-native-batch.md [hunk 1]
```diff
@@ -35,8 +35,8 @@ ingestion method.
 | Parallel indexing | Always parallel | Parallel if firehose is splittable | Always sequential |
 | Supported indexing modes | Replacing mode | Both appending and replacing modes | Both appending and replacing modes |
 | External dependency | Hadoop (it internally submits Hadoop jobs) | No dependency | No dependency |
-| Supported [rollup modes](http://druid.io/docs/latest/ingestion/index.html#roll-up-modes) | Perfect rollup | Best-effort rollup | Both perfect and best-effort rollup |
-| Supported partitioning methods | [Both Hash-based and range partitioning](http://druid.io/docs/latest/ingestion/hadoop.html#partitioning-specification) | N/A | Hash-based partitioning (when `forceGuaranteedRollup` = true) |
+| Supported [rollup modes](https://druid.apache.org/docs/latest/ingestion/index.html#roll-up-modes) | Perfect rollup | Best-effort rollup | Both perfect and best-effort rollup |
```
Comment (vogievetsky) [inline docs/content/ingestion/hadoop-vs-native-batch.md]: Why is this not a `/docs/...` link?

File: docs/content/ingestion/hadoop-vs-native-batch.md [hunk 2]
```diff
@@ -35,8 +35,8 @@ ingestion method.
...
+| Supported partitioning methods | [Both Hash-based and range partitioning](https://druid.apache.org/docs/latest/ingestion/hadoop.html#partitioning-specification) | N/A | Hash-based partitioning (when `forceGuaranteedRollup` = true) |
```
Comment (vogievetsky) [inline docs/content/ingestion/hadoop-vs-native-batch.md]: ditto

File: examples/bin/jconsole.sh [hunk 3]
```diff
@@ -18,7 +18,7 @@
 # under the License.
 ############################
 # This script downloads the appropriate log4j2 jars and runs jconsole with them as plugins.
-# This script can be used as an example for how to connect to a druid.io instance to
+# This script can be used as an example for how to connect to a druid instance to
```
Comment (vogievetsky) [inline examples/bin/jconsole.sh]: Capitalize Druid?

File: web-console/src/variables.ts [hunk 4]
```diff
@@ -21,10 +21,10 @@ export const LEGACY_OVERLORD_CONSOLE = '/console.html';
 
 export const DRUID_WEBSITE = 'http://druid.io';
```
Comment (vogievetsky) [inline web-console/src/variables.ts]: This one also?

File: docs/content/ingestion/hadoop-vs-native-batch.md (hunk 1 above)
Comment (clintropolis) [reply docs/content/ingestion/hadoop-vs-native-batch.md]: Hmm, good question. I just did a find and replace 🙃, will fix

File: web-console/src/variables.ts (hunk 4 above)
Comment (clintropolis) [reply web-console/src/variables.ts]: Nice, travis caught this one https://travis-ci.org/apache/incubator-druid/jobs/547030885#L1745

**Comment by vogievetsky** [comment]
Looking good at least for the WebConsole part

**Pull request merged**